cloudman/values.yaml

```diff
@@ -62,6 +62,89 @@ helmsman_config:
           jupyter_notebook_config.py: |
             c.NotebookApp.notebook_dir = '{{context.starting_dir}}'
 
+    nextcloud:
+      repo: cloudve
+      chart: nextcloud
+      display_name: File Management (NextCloud)
+      context:
+        oidc_client_secret: '{{.Values.oidc_clients.nextcloud.client_secret}}'
+        oidc_client_id: '{{ .Values.oidc_clients.nextcloud.client_id}}'
+        domain: '{{ .Values.global.domain }}'
+      template: |
+        ingress:
+          enabled: true
+          access_path: "/{{context.project.access_path}}/nextcloud"
+          hosts:
+            - name: {{context.domain}}
+              path: /{{context.project.access_path}}/nextcloud
+
+        nextcloud:
+          configs:
+            path.config.php: |
+              <?php
+              $CONFIG = array (
+                'overwriteprotocol' => 'https',
+                'overwritewebroot' => '{{context.project.access_path}}/nextcloud',
+                'overwritehost'     => '{{context.domain}}',
+                'overwrite.cli.url' => 'https://{{context.domain}}/{{context.project.access_path}}/nextcloud',
+                'skeletondirectory' => '',
+                'allow_user_to_change_display_name' => false,
+                'lost_password_link' => 'disabled',
+                'oidc_login_default_quota' => 1000000000,
```

Comment:
Please add a little context to what this means like discussed in slack to have for reference later

Reply:
`oidc_login_default_quota` optionally assigns a default storage quota to OIDC users if not provided one by Keycloak. The limit only concerns files stored in the Nextcloud home directory and does not factor in external storage mounts (e.g. `/gvl/`, S3, etc...). If a default quota is not set and no quota was provided by Keycloak, the user has no storage limits. That being said, it's not very useful at the moment since `/var/www/html/data` is not persistent so any data in the user's NC directory will be deleted.

More information on NC storage limits: https://docs.nextcloud.com/server/17/user_manual/files/quota.html
Instructions on setting up quotas in Keycloak from the NC OIDC README: https://github.com/pulsejet/nextcloud-oidc-login#usage-with-keycloak